code/modules/surgery/surgery.dm

```diff
@@ -44,15 +48,22 @@
 	if(replaced_by == /datum/surgery)
 		return FALSE
 
-	if(HAS_TRAIT(user, TRAIT_SURGEON) || (user.mind && HAS_TRAIT(user.mind, TRAIT_SURGEON)))
+	if(HAS_TRAIT(user, TRAIT_SURGEON) || user.mind && HAS_TRAIT(user.mind, TRAIT_SURGEON))
 		if(replaced_by)
 			return FALSE
 		else
 			return TRUE
+	//Grants the user innate access to all surgeries
+
+	if(HAS_TRAIT(user, TRAIT_ABDUCTOR_SURGEON) || user.mind && HAS_TRAIT(user.mind, TRAIT_ABDUCTOR_SURGEON))
+		if(replaced_by)
+			return FALSE
+		else if(!abductor_surgery_blacklist)
+			return TRUE
```

Comment:
Abductors should be able to perform this surgery if the operating table has it unlocked (If its in the blacklist, then instead of instantly returning, continue)